clients/src/test/java/org/apache/kafka/common/record/FileRecordsTest.java

```diff
@@ -295,25 +290,24 @@ public void testTruncateNotCalledIfSizeIsBiggerThanTargetSize() throws IOExcepti
             // expected
         }
 
-        EasyMock.verify(channelMock);
+        verify(channelMock, atLeastOnce()).size();
     }
 
     /**
      * see #testTruncateNotCalledIfSizeIsSameAsTargetSize
      */
     @Test
     public void testTruncateIfSizeIsDifferentToTargetSize() throws IOException {
-        FileChannel channelMock = EasyMock.createMock(FileChannel.class);
+        FileChannel channelMock = mock(FileChannel.class);
 
-        EasyMock.expect(channelMock.size()).andReturn(42L).atLeastOnce();
-        EasyMock.expect(channelMock.position(42L)).andReturn(null).once();
```

Comment:
Do we still need to verify that `channelMock.position` is called once?

Given that the old test works, `channelMock.position(...)` should be invoked. If we don't do `when(channelMock.position(42L)).thenReturn(null)`, what would be the return value if `channelMock.position(...)` is called?

It seems that channelMock.position() is called in the old code but not the new code. If so I am wondering which part changes this behavior.

Reply:
I removed that because it didn't seem relevant to this test. Mockito handles the `position` call correctly without us doing anything. With EasyMock, if not using a `Nice` mock, we have to specify every method called.

Reply:
I see. Sounds good.